Prac_06/guitar.py

```diff
@@ -0,0 +1,16 @@
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+        self.age = 0
+
+    def __str__(self):
+        return "{} ({}) : ${}".format(self.name, self.year, self.cost)
+
+    def get_age(self):
+        self.age = 2020 - self.year
+        return self.age
+
+    def is_vintage(self):
+        return self.get_age() >= 50
```

Comment:
Same as above, using a constant at the top of the code allows another user to understand what 50 means in this case.
`VINTAGE_YEAR = 50`